NebulaPatcher/Patches/Dynamic/UIMainMenu_Patch.cs

```diff
@@ -171,6 +171,18 @@ private static void OnJoinGameButtonClick()
             string ip = parts[0];
             int port;
 
+            //remove copy and paste mistakes and update the textbox to prevent user confusion in case of invalid ip address
+            ip = SanitizeIpAddressString(ip);
+            hostIPAdressInput.text = parts.Length == 1 ? ip : ip + ":" + parts[1];
+
+            if (!System.Net.IPAddress.TryParse(ip, out _))
+            {
+                //log, notify user and early return to prevent crash on invalid strings
+                InGamePopup.ShowWarning("Invalid connection string", "Note: This mod currently only accepts IPv4 Addresses.", "OK");
```

Comment:
Make sure the text in this message box is not clipped on low resolutions / high scaling, it's quite long

Reply:
I just tried this on 640x480 on my 1440p display, i don't recommend it, but at least it isn't clipping ;)
![grafik](https://user-images.githubusercontent.com/60819856/115295186-c12c4f80-a159-11eb-8283-85fd422e2147.png)

Reply:
Alright then, that's one concern alleviated!